users/stephen.ford.yaml

```diff
@@ -0,0 +1,72 @@
+---
+name: stephen.ford
+email: stephen.ford@digital.cabinet-office.gov.uk
+ARN: arn:aws:iam::622626885786:user/stephen.ford@digital.cabinet-office.gov.uk
+roles:
+- verify-admin
+- sandbox-sre
+hardware:
+  id: 9599499
+  type: yubikey
+github: smford
+teams:
+- re-gsp
+pub: |
+	-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
I think this block needs an additional space all the way down.

Reply:
cheers